src/ciphersuite/mod.rs

```diff
@@ -374,6 +374,11 @@ impl SignatureKeypair {
     pub fn get_public_key(&self) -> &SignaturePublicKey {
         &self.public_key
     }
+
+    /// Get the private and public key objects
+    pub fn as_tuple(self) -> (SignaturePrivateKey, SignaturePublicKey) {
```

Comment:
If we want to stick to Rust naming guidelines this should be `into_`.

Reply:
Right, good point!